iscript/src/iscript/autograph.py

```diff
@@ -500,17 +500,10 @@ def langpack_id(app):
     return id
 
 
-async def sign_langpacks(config, sign_config, all_paths):
-    """Signs langpacks that are specified in all_paths.
-
-    Raises:
-        IScriptError if we don't have any valid language packs to sign in any path.
-
-    """
+async def sign_langpacks(config, sign_config, all_paths, fmt):
+    """Signs langpacks that are specified in all_paths."""
     for app in all_paths:
-        app.check_required_attrs(["orig_path", "formats", "artifact_prefix"])
-        if not {"autograph_langpack"} & set(app.formats):
-            raise IScriptError(f"{app.formats} does not contain 'autograph_langpack'")
+        app.check_required_attrs(["orig_path", "artifact_prefix"])
```

Comment:
`formats` is dropped here because it's not used AFAICT.

Reply:
I guess formats is required anyway, by `filter_apps` / `get_langpack_format`?

Reply:
Yeah, it's required above `sign_langpacks` but not within it. (I guess I could arguably keep this...I don't really have strong feelings.)

Reply:
Removing it seems fine to me.